transaction/src/auth_hash.rs

```diff
@@ -152,6 +152,11 @@ impl TransactionPlan {
         for swap_claim in self.swap_claim_plans() {
             state.update(swap_claim.swap_claim_body(fvk).auth_hash().as_bytes());
         }
+        let num_clues = self.clue_plans.len() as u32;
```

Comment:
My bad because I sent you astray in Discord earlier, but the two places that need to match with respect to the clue keys are: https://github.com/penumbra-zone/penumbra/blob/main/transaction/src/auth_hash.rs#L189 and https://github.com/penumbra-zone/penumbra/blob/main/transaction/src/auth_hash.rs#L92

I think they do match for clue keys, so the stanza here adding the clue keys into the auth hash should stay after adding all the actions